File: leetcode/LengthofLastWord/lengthOfLastWord.java
public class Solution {
    public int lengthOfLastWord(String s) {
        // Start typing your Java solution below
        // DO NOT write main() function
        String arr[] = s.split("\\s+");
        int len = arr.length;
        return len == 0 ? 0 : arr[len - 1].length();
    }
}

File: leetcode/LengthofLastWord/lengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int size = s.size();
        if (!size) return 0;
        int begin = size - 1;
        while (begin >= 0 && s[begin] == ' ') begin--;
        int end = begin;
        while (end >= 0 && s[end] != ' ') end--;
        return begin - end;
    }
};
